crates/lib/src/cli.rs

```diff
@@ -52,6 +52,14 @@ use crate::store::{BootedOstree, ComposefsRepository, Storage};
 use crate::store::{BootedStorage, BootedStorageKind};
 use crate::utils::sigpolicy_from_opt;
 
+/// The result of a `bootc container inspect` command.
```

Comment:
I think let's put this in say spec.rs to make clearer it's public API, as distinct from the rust structs for CLI parsing?

Reply:
Moved!